numpy/random/tests/test_random.py

```diff
@@ -255,6 +255,18 @@ def test_random_integers_max_int(self):
         desired = np.iinfo('l').max
         np.testing.assert_equal(actual, desired)
 
+    def test_random_integers_deprecated(self):
```

Comment:
Yep, clearing the registry is a good safety measure to ensure that the warning is alway enabled despite previous errors/bad code.